### PR TITLE
Rework page structure to use flexbox so that sidebar columns extend with...

### PR DESCRIPTION
**Pull request by sg00dwin**
...out dynamically setting height

Adjustments to project-nav, primarily the label-selector so that it doesn't wrap and tighten up the look.

### CODE REVIEW

**Comment by sg00dwin** [comment]
@jwforres ready for review

File: assets/app/views/_project-page.html [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+  <div class="container-main">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-12" ng-transclude>
 
+        </div>
       </div>
     </div>
   </div>
+  <div class="sidebar-right">
```
Comment (jwforres) [inline assets/app/views/_project-page.html]: is this going to be visible, don't think we want it to be yet

File: assets/app/styles/_nav.less [hunk 2]
```diff
@@ -1,604 +0,0 @@
...
-.console-os .sidebar .nav.nav-sidenav > li.active > a:hover {background-color: #636b73;}
-.console-os .nav.nav-sidenav > li > a:hover {background-color: #737c85}
-
-.console-os .navbar-project,
-.console-os .nav-sidenav > li {border-bottom: 1px solid #44494e;}
-.console-os .nav-sidenav > li > a {color: #f2f3f4;}
- */
-
-
- .tile {
```
Comment (jwforres) [inline assets/app/styles/_nav.less]: just checking that all these styles were ok to delete, assuming these got moved to _tile.less at some point and were just duplicates?

File: assets/app/views/_project-page.html (hunk 1 above)
Comment (sg00dwin) [inline assets/app/views/_project-page.html]: It's set to display none by default and only shows when body class="show-sidebar-right" is applied.

File: assets/app/styles/_nav.less (hunk 2 above)
Comment (sg00dwin) [inline assets/app/styles/_nav.less]: yep, _tile.less has those.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**